dracut/repartition/endless-repartition.service

```diff
@@ -10,7 +10,7 @@ ConditionPathExists=/etc/initrd-release
 # wait for the root device to be ready, but avoid running at the same time
 # as fsck.
 After=dev-disk-by\x2dlabel-ostree.device
-Before=systemd-fsck@dev-disk-by\x2dlabel-ostree.service
+Before=systemd-fsck@dev-disk-by\x2dlabel-ostree.service systemd-fsck-root.service
```

Comment:
Couldn't we also remove the by-label/ostree fsck service ordering dependency, or is it still necessary?